src/support/d.ts

```diff
@@ -20,6 +22,20 @@ export function assignProperties(target: WNode | HNode, properties: WidgetProper
 }
 
 /**
+ * Creates a function which when placed in an expected render will call the `callback`.  If the `callback` returns `true`, the value
```

Comment:
Creates a function which when placed in an expected render will call the `callback` -> Creates a function which, when placed in an expected render, will call the `callback`